skimage/draw/draw.py

```diff
@@ -91,6 +91,23 @@ def ellipse(r, c, r_radius, c_radius, shape=None, rotation=0.):
         ((x * cos(alpha) + y * sin(alpha)) / x_radius) ** 2 +
         ((x * sin(alpha) - y * cos(alpha)) / y_radius) ** 2 = 1
 
+
+
+    Note, the positions of `ellipse` without specified `shape` can
+    have also negative values as it is correct in the plane. On the other hand
+    using these ellipse positions for an image afterwords may leads to appearing
+    on the other isde of image because ``image[-1, -1] = image[end-1, end-1]``
```

Comment:
`isde` would be `side`, right? :)
Also, could you put a comma between `image` and `because`?